Makefile

```diff
@@ -162,6 +155,7 @@ $(CRD_JSONNET_FILES): $(GOJSONTOYAML_BINARY) $(CRD_YAML_FILES)
 	cat example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml     | gojsontoyaml -yamltojson > jsonnet/prometheus-operator/podmonitor-crd.libsonnet
```

Comment:
Here we should use ```$(GOJSONTOYAML_BINARY)``` instead of gojsontoyaml.

Reply:
Command was removed during rebased :shrug: